### PR TITLE
ButtonSet Docs: Adds `description` and `caption` for `ButtonSet` component

### PR DESCRIPTION
**Pull request by MelSumner**
### :pushpin: Summary

If merged, this PR adds `description` and `caption` to the `ButtonSet` component documentation. 
### :hammer_and_wrench: Detailed description

- description: Provides consistent layout and spacing for a set of buttons.
- caption: Group of buttons.

***

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

File: website/docs/components/button-set/index.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ---
 title: ButtonSet
+description: Provides consistent layout and spacing for a set of buttons.
+caption: Group of buttons.
```
Comment (heatherlarsen) [inline website/docs/components/button-set/index.md:4]: I'm not sold on this caption being so short, but I'll have to see it along with the rest of the cards once these are merged up. 

```suggestion
caption: A set of buttons.
```

**Review by heatherlarsen — CHANGES_REQUESTED** [review]

File: website/docs/components/button/index.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ---
 title: Button
+description: An interactive element that initiates an action or event, like a form submission.
```
Comment (KristinLBradley) [inline website/docs/components/button/index.md:3]: Suggestion:
"like a form submission" 
=>
"such as a form submission"

File: website/docs/components/card/index.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 ---
 title: Card
+description: A block container that provides styling for elevation, border and background.
```
Comment (KristinLBradley) [inline website/docs/components/card/index.md:3]: Add oxford comma:
"A block container that provides styling for elevation, border and background."
=>
"A block container that provides styling for elevation, border, and background."

File: website/docs/components/button/index.md (hunk 2 above)
Comment (MelSumner) [inline website/docs/components/button/index.md:3]:
```suggestion
description: An interactive element that initiates an action or event, such as a form submission.
```

File: website/docs/components/card/index.md (hunk 3 above)
Comment (MelSumner) [inline website/docs/components/card/index.md:3]:
```suggestion
description: A block container that provides styling for elevation, border, and background.
```

File: website/docs/components/card/index.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
...
+caption: A block container that provides styling for elevation, border and background.
```
Comment (MelSumner) [inline website/docs/components/card/index.md:4]:
```suggestion
caption: A block container that provides styling for elevation, border, and background.
```

File: website/docs/components/dropdown/index.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
 ---
 title: Dropdown
+description: Displays a list (typically of actions or links) through a toggle and is identifiable by the chevron icon in the button.
```
Comment (KristinLBradley) [inline website/docs/components/dropdown/index.md:3]: Would this be clearer do you think?
"Displays a list (typically of actions or links) revealed by a toggle button. Identifiable by the chevron icon in the button."

Comment (MelSumner) [inline website/docs/components/dropdown/index.md:3]:
```suggestion
description: Displays a list (typically of actions or links) revealed by a toggle button. Identifiable by the chevron icon in the button.
```

Comment (MelSumner) [reply website/docs/components/dropdown/index.md:3]: Great idea! I'll change it.

**Pull request merged**